javascript-methods/javascript-methods-quiz-notes.md

```diff
@@ -0,0 +1,65 @@
+# javascript-methods-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- Why do we log things to the console?
+we log things to the console to check the output and that helps us learn if our codes generate the outcome we intended.
+- What is a method?
+method is a function that is attached to a javascript object.
+
+- How is a method different from any other function?
+method is attached to a js object, other functions can be independent.
+
+- How do you remove the last element from an array?
+using the pop() method of the array object
+
+- How do you round a number down to the nearest integer?
+using the floor() method of the math object
+
+- How do you generate a random number?
+using the random() method of the math object
+
+- How do you delete an element from an array?
+using the splice method of the array, pass in the first argument as the starting index, and 2nd as the number of items to delete from that index).
+
+- How do you append an element to an array?
+using the push() method or unshift() method
+- How do you break a string up into an array?
+using the split() method and pass a character where you want to split the string.
+- Do string methods change the original string? How would you check if you weren't sure?
+they dont. I check by plugging in the original string in the console after applying the methods.
+- Roughly how many string methods are there according to the MDN Web docs?
+about 47
+- Is the return value of a function or method useful in every situation?
+yes?
+- Roughly how many array methods are there according to the MDN Web docs?
+unsure
```

Comment:
If you go to the MDN docs for a method such as [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) you will see in the left hand column a **Methods** section where you could get an approximate count.